### ISSUE TITLE
Images on /new have no alt text

### ISSUE THREAD

**Issue by rraue** [mozilla/bedrock#9331]
## Description

The new images on /new have no alt text what is an issue for accessibility and SEO:

![Screenshot 2020-08-26 at 13 52 41](https://user-images.githubusercontent.com/36700618/91300412-84af7c00-e7a3-11ea-92a3-7d4edcc41663.png)

**Comment by soulhunter10737** [comment]
Where it is located? I would like to take this

**Comment by ishantd** [comment]
Greetings @rraue , I have added alt text for all the images on the /new page and created a pull request, please review. #9477

**Comment by nox1134** [comment]
Hi @alexgibson! If no one is working on this issue, I'd like to take up this issue and work on it.
Could you please assign the issue to me?

**Comment by janbrasna** [comment]
While there are handy strings already translated in [firefox/new/desktop.ftl](https://github.com/mozilla/bedrock/blob/main/l10n/en/firefox/new/desktop.ftl) that could be cheaply recycled here, I'd say they'd do more harm than anything. I've quickly checked all the mentioned images, and unless I've missed something they are completely decorative, with the meaning covered by the adjacent text. Therefore empty alt is correct: https://www.w3.org/WAI/tutorials/images/decorative/

Take the "Screenshot" section as an example. It says:

> Screenshots
> Grab a high-resolution image of anything online with our screenshot tool built right in the browser

I could easily add an alt="Built-in screenshot tool" from elsewhere in the page, but that would read that section for assistive tech as:

> Built-in screenshot tool
> Screenshots
> Grab a high-resolution image of anything online with our screenshot tool built right in the browser

That makes it only more busy and repetitive.

It may need another opinion and xcheck of the images for any meaning not covered by text, but I'm leaning towards wontfix.

**Comment by alexgibson** [comment]
Agree, adding alt text here would not provide anything useful for screen reader users here, closing.
